### ISSUE TITLE
rename? increased transcript length by 3' extension

### ISSUE THREAD

**Issue by ValWood** [pombase/fypo#3216]
call "increased transcriptional readthrough"?

increased transcript length by 3' extension

**Comment by mah11** [comment]
Well, that wouldn't mean exactly the same thing, because the current wording is neutral with respect to transcription versus extra poly(A) addition. But it was added for mutants in PMID:22582262 that don't have a poly(A) site (see #1301), where extra length would be due to more transcription. And despite being added upon request & for a paper, it doesn't have any annotations on the term page, so I think we are free to rejig it (and the current name is kind of horrible).

**Comment by ValWood** [comment]
hmm, it's used in the session...curs/e3c8313e9bd1dabc/ro

**Comment by ValWood** [comment]
but def not on the page...

**Comment by ValWood** [comment]
I think for the term more neutral wording would be more understandable....

**Comment by mah11** [comment]
The current wording _is_ the neutral (albeit ugly) wording: it doesn't specify whether the extra length was added by overzealous transcription or excess polyadenylation. I'm saying it would be OK to make it specify transcription, because that's consistent with the only way it's been used in annotations (I've since checked them all via the downloadable PHAF).

I think the missing-from-pages issue is a config thing that I can fix.
